static/js/components/material/Button.js

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+
+export default class Button extends React.Component {
+  render() {
+    const { children, className, ...otherProps } = this.props;
+
+    return <button
+      className={className ? `mdc-button ${className}` : 'mdc-button'}
```

Comment:
I like this pattern better because it avoid a trailing space (`'mdc-button '`) which admittedly is super fussy, so if you feel strongly that that way is better I'm open to changing it.